src/Domain/CoordinateMaps/Frustum.cpp

```diff
@@ -511,18 +562,28 @@ void Frustum::pup(PUP::er& p) {
     p | sphericity_;
     p | radius_;
     p | phi_;
+    if (version < 2) {
+      bool with_projective_map = false;
+      p | with_projective_map;
```

Comment:
This won't work because the location of the bool in the data stream is after `is_identity_`, not here at the end. Move up to where you deleted `p | with_projective_map_`.